### DISCUSSION TITLE
Add Library to PyPi

### DISCUSSION THREAD

**Discussion by Quadrob** [rlehfeld/robotframework-async-keyword#2] [General]
Could you add this library to the official Python Package Index (https://pypi.org/)? 🙂

**Comment by rlehfeld** [comment]
Yes, I already started looking into this. This is definitely something that I have in my mind.

**Reply by rlehfeld** [reply]
Done, but I had to rename the repository so it does not clash with the original

**Reply by rlehfeld** [reply]
documentation is also updated accordingly. Pipelines added so this is an automatic process with every new release that is created.

**Reply by rlehfeld** [reply]
See announcement

**Reply by Quadrob** [reply]
Oh nice awesome thanks René 🙂

**Reply by rlehfeld** [reply]
I still have to see how I can improve the description of the package on pypi. It currently has no content :-( but I am confident that this is just a matter of time.

**Reply by rlehfeld** [reply]
there you go. Now also with description in pypi

**Reply by rlehfeld** [reply]
and now I am done. There were some pipeline warnings which are also now resolved.
